.github/workflows/build-deploy-forms.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy Formio forms
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment the formio is deployed."
+        required: true
+        type: environment
+      formioTag:
+        description: "Formio tag, please refer https://github.com/formio/formio/tags to deploy the appropriate tag."
+        type: string
+        required: true
+        default: "v2.5.3"
+  workflow_call:
```

Comment:
I would say that it is not needed right now.
It is only needed it this workflow will be invoked by another workflow, which is not the case right now.